Docker/entrypoint.sh

```
@@ -28,11 +28,36 @@
+    echo "Model files found in /data/model"
+    # Set read permissions for model files
+    chmod -R a+r /data/model/*.tflite 2>/dev/null || true
+    # Ensure directory is executable (browsable)
+    chmod a+x /data/model
+else
+    echo "Warning: No model files found in /data/model"
+    echo "BirdNET-Go will look for models in standard paths"
+fi
+
+# Check if user has custom model path configured via environment variable
+if [ ! -z "$BIRDNET_MODELPATH" ]; then
```

Shellcheck (0.10.0)

[style] Use -n instead of ! -z.

(SC2236)